omod/src/main/webapp/portlets/login.jsp

```diff
@@ -41,8 +41,7 @@
 				<input type="hidden" name="redirect" value="" />
 			</c:otherwise>
 		</c:choose>
-		
-		<input type="hidden" name="refererURL" value='<request:header name="referer" />' />
+		<input type="hidden" name="refererURL" value='<c:out value="${pageContext.request.getHeader('referer')}"/>' />
```

Comment:
Does this mean that the XSS attack came via the referer param?

Reply:
Yes